File: QCTransition/QCTransition/ViewController.m
//
//  ViewController.m
//  QCTransition
//
//  Created by 程庆春 on 23/11/2016.
//  Copyright © 2016 qiuncheng.com. All rights reserved.
//

#import "ViewController.h"
#import "TransitionAnimator.h"

@interface ViewController () <UINavigationControllerDelegate>

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    self.navigationController.delegate = self;
//    self.navigationController.navigationBar.delegate = self;
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (id<UIViewControllerAnimatedTransitioning>)navigationController:(UINavigationController *)navigationController animationControllerForOperation:(UINavigationControllerOperation)operation fromViewController:(UIViewController *)fromVC toViewController:(UIViewController *)toVC {
    return [TransitionAnimator new];
}

@end

File: SwifterSwiftDemo/SwifterSwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwifterSwiftDemo
//
//  Created by 程庆春 on 2016/12/13.
//  Copyright © 2016年 qiuncheng.com. All rights reserved.
//

import UIKit
import SwifterSwift

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let str = "hello world"
        print(str.camelCased)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}
